### DISCUSSION TITLE
Stuck on "Waiting for first pageview" even though pageviews are recorded in Clickhouse.

### DISCUSSION THREAD

**Discussion by acidbiscuit** [plausible/analytics#1819] [Self-Hosted Support] [answered]
I'm running Plausible via docker-compose on DO and everything appears to be working correctly up to the point when dashboard is stuck on "Waiting for first pageview on ...".  
Manual check with `docker exec` in running Clickhouse container shows that `pageview` events are recorded immediately, but dashboard status polling returns `WAITING`. 

No errors have been logged in any of running containers. I've tried changing ports, switching env variables, connecting containers with docker `network`, but no luck.

Any suggestions on how to solve/debug the issue?

The only difference with Plausible docker-compose example is that I'm passing `env` via docker-compose, instead of dedicated `plausible-conf.env` file. But the test with providing env as a file did not make a difference.

Attaching relevant screenshots and docker-compose file.

1. [docker-compose.yaml](https://gist.github.com/acidbiscuit/e1a0e3f38e2e9edce29213fc647af761)
2. Clean Plausible start up console - [screenshot](https://i.imgur.com/E6pwqCQ.png)
3. Events recorded to Clickhouse - [screenshot](https://i.imgur.com/5INzJ1P.png)

**Comment by ukutaht** [comment, marked as answer]
Try removing `https://` from the `domain` field. Just use `aidua.today` instead of `https://aidua.today`

**Reply by acidbiscuit** [reply]
That worked. Thank you for taking your time to look into the problem, @ukutaht

**Reply by ukutaht** [reply]
Cheers! Enjoy using Plausible
